apps/cookbook/src/app/examples/list-load-on-demand-example/list-load-on-demand-example.component.ts

```diff
@@ -1,9 +1,10 @@
 import { Component } from '@angular/core';
 import { of } from 'rxjs';
 
-import { BaseListComponent } from '../base-list.component';
 import { LoadOnDemandEvent } from '@kirbydesign/designsystem';
 
+import { BaseListComponent } from './base-list.component';
```

Comment:
Why the need for the duplicated `BaseListComponent? 🤔

Reply:
I wanted to keep each folder with examples self-contained 🎁

The `list-load-on-example-component` and the examples that are now found in `list-example/components/...`  initially pointed to the same `base-list.component`, as both were contained in the same parent folder. When splitting up the original `list`-folder, i did not want the `list-load-on-demand` example to depend on files in the `list-example`-folder. Therefore i created a duplicate.

It might however make more sense to just copy the data in `list-load-on-demand-example/base-list.component` directly into the `list-load-on-demand-example.component`-file. Might just muddy the file a bit, but it is done like that in the other list examples that are not located in the `list-example`-folder.

Reply:
Consider moving the file to a shared location instead 🤔

Reply:
As discussed on Slack with the new commit duplicates has been removed and baseListComponent is now stored in a list-shared folder 🗂 Should resolve comment.